test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -54,9 +54,19 @@ public void CheckToOtlpLogRecordStateValues()
 
             Assert.NotNull(otlpLogRecord);
             Assert.Equal("Hello from tomato 2.99.", otlpLogRecord.Body.StringValue);
-            Assert.Contains("name", otlpLogRecord.Attributes.ToString());
-            Assert.Contains("tomato", otlpLogRecord.Attributes.ToString());
-            Assert.Contains("{OriginalFormat}", otlpLogRecord.Attributes.ToString());
+            Assert.Equal(3, otlpLogRecord.Attributes.Count);
+
+            var attribute = otlpLogRecord.Attributes[0];
+            Assert.Equal("name", attribute.Key);
+            Assert.Equal("tomato", attribute.Value.StringValue);
+
+            attribute = otlpLogRecord.Attributes[1];
+            Assert.Equal("price", attribute.Key);
+            Assert.Equal(2.99, attribute.Value.DoubleValue);
+
+            attribute = otlpLogRecord.Attributes[2];
+            Assert.Equal("{OriginalFormat}", attribute.Key);
```

Comment:
For next PR, I will be tweaking exporter to special case this, as that is the behavior of the commonly used logging pattern with "message-templates", when using the ILogger.Log extension methods.

This PR is just preparing the tests to see if this is a good approach to test things..